cmd/minishift/cmd/addon/addon_uninstall_test.go

```diff
@@ -57,25 +59,22 @@ func Test_uninstall_with_enable_flag_works(t *testing.T) {
 	defer cli.TearDown(tmpMinishiftHomeDir, tee)
 
 	addOnManager := GetAddOnManager()
-	if len(addOnManager.List()) != 0 {
-		t.Fatal(fmt.Sprintf("There should be no add-ons installed. Got : %v", addOnManager.List()))
-	}
+
+	assert.Len(t, addOnManager.List(), 0)
```

Comment:
assert.Empty! I recommend you have a look at all public asserts in _assertions.go_ and get a feel for them.

Reply:
Sure, i will go through them all, and try to use more appropriate assertions.